waiter/src/waiter/token.clj

```diff
@@ -157,6 +169,25 @@
                   token-set (->> tokens (map :token) set)]
               (kv/store kv-store owner-key token-set))))))))
 
+;; A token store which is backed by a key-value store.
+(defrecord KeyValueTokenStore [clock synchronize-fn kv-store]
+
+  TokenStore
+
+  (fetch [_ token include-deleted?]
+    (sd/token->token-description kv-store token :include-deleted include-deleted?))
```

Comment:
Inline `sd/token->token-description`

Reply:
`sd/token->token-description` and `sd/token->service-description-template` are closely related.